roadmaps/configuration.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: contribute
+title: Bazel Configurability Roadmap
+---
+<style>
+  .padbottom { padding-bottom: 10px; }
+  .etabox {
+    background: #EFEFEF;
+    color: #38761D;
+    font-size: 15px;
+    font-weight: bold;
+    display: inline;
+    padding: 6px;
+    margin-right: 10px;
+  }
+</style>
+
+# Bazel Configurability 2018 Roadmap
+*Last verified: Wednesday, June 6, 2018* ([update history]
+(https://github.com/bazelbuild/bazel-website/commits/master/roadmaps/configuration.md))
+
+This page describes committed plans to advance Bazel's configuration and
+multiplatform support in 2018.
+
+Dates are approximate based on our best understanding of problem complexity
+and developer availability. ETAs will change, but we'll keep them refreshed and
+current.
+
+Have a bug or feature request? File a [GitHub issue
+](https://github.com/bazelbuild/bazel/issues/new).
+
+Questions or suggestions? Email
+<a href="mailto:bazel-discuss@googlegroups.com">bazel-discuss@googlegroups.com</a>.
+
+
+* [Goal](#goal)
+* [Roadmap](#roadmap)
+  * [Platforms](#platforms)
+  * [User-Defined Configuration](#user-defined-configuration)
+  * [Correctness and Speed](#correctness_and_speed)
+
+
+<a name="goal"></a>
```

Comment:
I believe this isn't needed, and anchors are created automatically from headings in our Markdown processor.

Reply:
Ah, it turns out auto-anchoring was set for [bazel](https://github.com/bazelbuild/bazel/blob/708b957f7c37fd83410588d8ef3792135abd4d8d/site/_includes/footer.html#L10-L15) but not [bazel-website](https://github.com/bazelbuild/bazel-website/blob/f8b9662c68eb022bb59c43e8c9c60b5a1683827b/_includes/footer.html).

Added it there too.